### PR TITLE
Add KSoft.Si Lyrics

### PR DESCRIPTION
**Pull request by johan-naizu**
KSoft.Si is a simple and powerful lyrics API

- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by vineyrawat** [comment]
This is really cool API

**Review by fintzd — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -707,6 +707,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Jamendo](https://developer.jamendo.com/v3.0/docs) | Music | `OAuth` | Yes | Unknown |
 | [JioSaavn](https://github.com/cyberboysumanjay/JioSaavnAPI) | API to retrive song information, album meta data and many more from JioSaavn | No | Yes | Unknown |
 | [KKBOX](https://developer.kkbox.com) | Get music libraries, playlists, charts, and perform out of KKBOX's platform | `OAuth` | Yes | Unknown |
+| [KSoft.Si](https://api.ksoft.si) | API to get lryics for songs | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:710]: This project is not just a lyrics API. This project contains other APIs, as you can see in the image below:

ref: https://api.ksoft.si/
image

Therefore, this link (https://api.ksoft.si) does not fit in the music section. The lyrics API can be found at this link: https://docs.ksoft.si/api/lyrics-api

- You can update the link and just add the lyrics API
```suggestion
| [KSoft.Si](https://docs.ksoft.si/api/lyrics-api) | API to get lryics for songs | `apiKey` | Yes | Unknown |
```
- or add all 4 APIs in the sections that best match
   - Image API: https://docs.ksoft.si/api/images-api
   - Bans API: https://docs.ksoft.si/api/bans-api
   - Kumo API: https://docs.ksoft.si/api/kumo-api
   - Lyrics API: https://docs.ksoft.si/api/lyrics-api

- or just add https://api.ksoft.si in a section that best matches (my suggestion is add in development section)

What do you prefer?

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @johan-naizu 😄 

Can you resolve this review, please?

File: README.md (hunk 1 above)
Comment (johan-naizu) [reply README.md]: Made changes

File: README.md [hunk 2]
```diff
@@ -707,6 +707,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Jamendo](https://developer.jamendo.com/v3.0/docs) | Music | `OAuth` | Yes | Unknown |
 | [JioSaavn](https://github.com/cyberboysumanjay/JioSaavnAPI) | API to retrive song information, album meta data and many more from JioSaavn | No | Yes | Unknown |
 | [KKBOX](https://developer.kkbox.com) | Get music libraries, playlists, charts, and perform out of KKBOX's platform | `OAuth` | Yes | Unknown |
+| [KSoft.Si](https://docs.ksoft.si/api/lyrics-api) | API to get lryics for songs | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:710]: Can you fix this, please?

```suggestion
| [KSoft.Si Lyrics](https://docs.ksoft.si/api/lyrics-api) | API to get lyrics for songs | `apiKey` | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Can you resolve this new review, please?

**Comment by matheusfelipeog** [comment]
When finished, can you [squashed](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) all the changes into a single commit, please?

**Comment by johan-naizu** [comment]
Done

**Review by matheusfelipeog — APPROVED** [review]
Perfect, thanks for solving this @johan-naizu ❤️

**Pull request merged**